static/app/views/performance/trends/changeExplorerUtils/metricsTable.tsx

```diff
@@ -0,0 +1,413 @@
+    <SortLink
+      title={column.name}
+      align={align}
+      direction={undefined}
+      canSort={false}
+      generateSortLink={() => undefined}
+    />
+  );
+}
+
+export function renderBodyCell(
+  column: GridColumnOrder<MetricColumnKey>,
+  dataRow: TableDataRow
+) {
+  let data = '';
+  let color = '';
+  if (column.key === 'change') {
+    if (
+      dataRow[column.key] === '0%' ||
+      dataRow[column.key] === '+NaN%' ||
+      dataRow[column.key] === '-'
+    ) {
+      data = '-';
+    } else if (dataRow[column.key].charAt(0) !== '-') {
+      color = theme.red300;
+      data = '+' + dataRow[column.key];
+    } else {
+      color = theme.green300;
+      data = dataRow[column.key];
+    }
+  } else {
+    data = dataRow[column.key];
+  }
+
+  return (
+    <Container data-test-id={'pce-metrics-chart-row-' + column.key}>
+      <ExplorerText
+        data-test-id={'pce-metrics-text-' + column.key}
+        align={column.key !== 'metric' ? 'right' : 'left'}
+        color={color}
+      >
+        {data}
+      </ExplorerText>
+    </Container>
+  );
+}
+
+function getQueryParams(
+  startTime: string,
+  endTime: string,
+  fields: AggregationKeyWithAlias[],
+  query: string,
+  dataset: DiscoverDatasets,
+  organization: Organization,
+  eventView: EventView,
+  transactionName: string,
+  location: Location
+) {
+  const newLocation = {
+    ...location,
+    start: startTime,
+    end: endTime,
+    statsPeriod: undefined,
+    dataset,
+    sort: undefined,
+    query: {
+      query: `event.type: ${query} transaction: ${transactionName}`,
+      statsPeriod: undefined,
+      start: startTime,
+      end: endTime,
+    },
+  };
+
+  const newEventView = getEventViewWithFields(
+    organization,
+    eventView,
+    startTime,
+    endTime,
+    fields,
+    query,
+    transactionName,
+    dataset
+  );
+
+  return {
+    eventView: newEventView,
+    location: newLocation,
+    orgSlug: organization.slug,
+    transactionName,
+    transactionThresholdMetric: TransactionThresholdMetric.TRANSACTION_DURATION,
+    options: {
+      refetchOnWindowFocus: false,
+    },
+  };
+}
+
+type MetricColumnKey = 'metric' | 'before' | 'after' | 'change';
+
+type MetricColumn = GridColumnOrder<MetricColumnKey>;
+
+type TableDataRow = Record<MetricColumnKey, any>;
+
+const MetricColumnOrder = ['metric', 'before', 'after', 'change'];
+
+export const COLUMNS: Record<MetricColumnKey, MetricColumn> = {
+  metric: {
+    key: 'metric',
+    name: t('Metric'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+  before: {
+    key: 'before',
+    name: t('Before'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+  after: {
+    key: 'after',
+    name: t('After'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+  change: {
+    key: 'change',
+    name: t('Change'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+};
+
+const COLUMN_TYPE: Record<MetricColumnKey, ColumnType> = {
+  metric: 'string',
+  before: 'duration',
+  after: 'duration',
+  change: 'percentage',
+};
```

Comment:
We typically put constants and types like that at the top of the file below the imports